packages/lib-user/src/components/shared/ProfileHeader/ProfileHeader.js

```diff
@@ -45,12 +49,12 @@ TitledStat.propTypes = {
 
 function ProfileHeader ({
   avatar = '',
-  classifications = 0,
-  contributors = 0,
+  classifications = undefined,
+  contributors = undefined,
   displayName = '',
-  hours = 0,
+  hours = undefined,
   login = '',
-  projects = 0,
+  projects = undefined,
   screenSize = 'medium'
```

Comment:
NB suggestion, but in the places where you're using `withResponsiveContext()`, you can grab `size` directly from Grommet's ResponsiveContext like this example:
https://github.com/zooniverse/front-end-monorepo/blob/fd8b985db600e725a77bb376a0981d841ec26cfd/packages/app-project/src/screens/ProjectHomePage/ProjectHomePage.js#L16

Definitely not required, but I think generally HOC's named `with` were originally for Class components.